### DISCUSSION TITLE
Should I add .smart-connections to .gitignore?

### DISCUSSION THREAD

**Discussion by dsebastien** [brianpetro/obsidian-smart-connections#528] [Q&A] [answered]
I'm synchronizing my Obsidian vault with a private git repo on GitHub and using it on multiple devices.
Should I include or ignore the .smart-connections folder contents?
I wonder what the pros and cons are.

**Comment by brianpetro** [comment, marked as answer]
Hey @dsebastien 

In short, yes, you should add it to `.gitignore`.

The embedding files in the `.smart-connections` folder  are large and tracking the frequent changes in a version control would have no future benefits.

Thanks for asking
🌴

**Reply by andrexibiza** [reply]
Hey Brian, FYI I was having a really persistent issue with ALL of my embeddings refreshing every time I open Obsidian. I couldn't figure it out for a while. I ended up creating a new embeddings folder called "EmbeddingsData" and now it works without a hitch across my PC and laptop. Wanted to make you aware in case nobody's had this issue yet. I still don't understand why the default .smart-connections folder wasn't working. Thanks!

**Reply by brianpetro** [reply]
That is really good to know! Thanks for sharing. There are some other threads I'm going to share this with right now!

🌴

**Reply by vdruts** [reply]
Do you mean a new .smart-connections folder?

**Reply by Hopsakee** [reply]
Thanks for sharing this. I have the same 're-embedding issue'.

But when I choose a different folder, I regularly get this question :

![image](https://github.com/brianpetro/obsidian-smart-connections/assets/28975522/7d4348ad-2964-442a-a62f-1b58d03233e9)

I removed the EmbeddingsData folder from Obsidian sync and excluded it from the folders that SmartConnections should embed.

Any idea what I can do to solve this?

**Reply by brianpetro** [reply]
@Hopsakee I've never seen that dialog before. Is that an Obsidian native dialog? 

I wonder if for whatever reason that's happening may also be cause for the re-embedding issue 🤔

**Reply by Hopsakee** [reply]
Yes. It is native to obsidian. I also have never seen it except for now. And what makes it even more awkward. If I just ignore this, click cancel, it just soldiers on in the original embeddingsfile. If I give it the same name as the embeddingsfile, it creates a new file with the suffix `_1`.

Curious what happens once the embedding is complete.

I also replaced my complete vault to my home folder on windows instead of `c:\Obsidian\Personal` to make sure it's not some kind of permissions issue.
Didn't help.

I will let you know if I get any more of those dialogs once the embedding finished.
